flake.nix

```
@@ -44,15 +44,15 @@
         homeManagerConfiguration = { configuration, system, homeDirectory
           , username, extraModules ? [ ], extraSpecialArgs ? { }
           , pkgs ? builtins.getAttr system nixpkgs.outputs.legacyPackages
-          , check ? true, stateVersion ? "20.09" }@args:
+          , check ? true, stateVersion ? "20.09", useNixpkgsModule ? false
```

```suggestion
          , check ? true, stateVersion ? "20.09", useNixpkgsModule ? true
```

Let's toggle this to false by default for the sake of backward compatibility.